### PR TITLE
Update DIVD-2021-00021.md

### PR DESCRIPTION
**Pull request by Lennaert89**
case closed

### CODE REVIEW

**Review by Ralphhorn — APPROVED** [review]
lgtm

File: _cases/2021/DIVD-2021-00021.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ timeline:
   event:  "DIVD notified about patch by vendor."
 - start: 2022-03-01
   end:
-  event: "DIVD sent out a first batch of notifications."
```
Comment (MrSeccubus) [inline _cases/2021/DIVD-2021-00021.md:35]: Missing a case closed event

Comment (MrSeccubus) [reply _cases/2021/DIVD-2021-00021.md:35]: Also the end date isn;t set on line 19.

**Review by MrSeccubus — CHANGES_REQUESTED** [review]

**Review by MrSeccubus — APPROVED** [review]
Thanks for the changes.

**Pull request merged**